source/features/anonymize-tweets.js

```diff
@@ -0,0 +1,7 @@
+export default function () {
+	$('.FullNameGroup .fullname').text('Jane Doe');
+	$('.js-retweet-text b').text('Jane Doe');
+	$('.UserBadges').hide();
+	$('.username').hide();
```

Comment:
These two can be done call.
```suggestion
	$('. UserBadges, .username').hide();
```